client/css/style.css

```diff
@@ -587,11 +587,8 @@ button {
 }
 
 #main {
-	background: #fff;
```

Comment:
Because each `window` is taking care of its own `background-color`. And as far as I remember, it's necessary to remove this because of the radius being moved to `.window` instead of `#main` (letting us get rid of that ugly, half-working `overflow: hidden`.

Does that make more sense now?